src/main/java/com/solutions/shop/model/Product.java

```diff
@@ -35,6 +35,15 @@ public class Product {
     @NotBlank
     private String category;
 
+    public Product(Integer productId, String name, String description, BigDecimal price, String brand, String category) {
```

Comment:
lombok? You could use here @RequiredArgsConstructor annotation

Reply:
I want to have the Constructor without shoppingCart. Admin should create all products independently.

Reply:
Lombok @RequiredArgsConstructorc generates a constructor with required arguments. Required arguments are uninitialized final fields and fields with constraints such as @NonNull